lib/pagination.js

```diff
@@ -0,0 +1,51 @@
+/*
+Create a list of pagination pages. Returns an array of pages, possibly with
+a `null` break item:
+
+e.g. Pagination.pages({ currentPageNumber: 0, totalPages: 10, createPageLink })
+[
+  { pageNumber: 0, isCurrent: true, link: 'http://...' },
+  { pageNumber: 1, isCurrent: false, link: 'http://...' },
+  { pageNumber: 2, isCurrent: false, link: 'http://...' },
+  { pageNumber: 3, isCurrent: false, link: 'http://...' },
+  null,
+  { pageNumber: 8, isCurrent: false, link: 'http://...' },
+  { pageNumber: 9, isCurrent: false, link: 'http://...' }
+]
+
+`createPageLink` is a function that is passed the page object
+{ pageNumber, isCurrent } and should return the URL to that page.
+*/
+exports.pages = ({ currentPageNumber, totalPages, maxLinks = 6, createPageLink = null }) => {
+  maxLinks = maxLinks || 6; // Temporary fix to make tests work
```

Comment:
This is because istanbul doesn't yet support defaults for destructured parameters - https://github.com/gotwarlost/istanbul/issues/613